packages/ui/src/components/Button/Counter.tsx

```diff
@@ -1,4 +1,9 @@
 import { DetailedHTMLProps, HTMLAttributes, forwardRef } from "react";
+import { tv } from "tailwind-variants";
+
+const counter = tv({
+  base: "ease :ont-bold ml-2 rounded-md p-2 px-1 text-sm/[1] transition-all duration-200",
```

Comment:
Should the 
`:ont-bold`
be
`font-bold` or `:font-bold`

Reply:
I noticed loss of bolding here

![image](https://github.com/user-attachments/assets/1421b1d0-6c9b-47b3-b28a-dd32f97a4a86)

Reply:
Yup, also found another typo [fix typos in classes](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13654/commits/a08f5620003fc34112545dffd4cddc6c4c2b66b1)